engine/src/main/scala/cromwell/services/MetadataServiceActor.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.services
+
+import akka.actor.Actor
+import cromwell.engine.workflow.WorkflowManagerActor.WorkflowMetadata
+import cromwell.engine.{ExecutionEventEntry, WorkflowId}
+import cromwell.engine.backend.JobKey
+import cromwell.services.MetadataServiceActor._
+import cromwell.webservice.WorkflowMetadataResponse
+import org.joda.time.DateTime
+import wdl4s.values.WdlValue
+
+object MetadataServiceActor {
+
+  sealed trait MetadataEvent[V] {
+    def workflowId: WorkflowId
+    def jobKey: JobKey
```

Comment:
disregard above. My brief dalliance with worrying about scoping these services to backend calls is on hiatus